tests/units/test_thread_safety.rb

```diff
@@ -24,7 +24,7 @@ def test_git_init_bare
     threads.each(&:join)
 
     dirs.each do |dir|
-      Git.bare("#{dir}/.git").ls_files
+      Git.bare(dir).ls_files
```

Comment:
`Git.init` should create the new, bare repository in the directory given by `dir` and not in the `.git` subdirectory of `dir`. This was a bug.